installing/installing-mirroring-installation-images.adoc

```diff
@@ -65,6 +65,37 @@ In production environments, add the required images to a registry in your restri
  modules/installation-performing-disconnected-mirror-without-registry.adoc[leveloffset=+2]
 ////
 
+[id="mirror-registry"]
+== Mirror registry for {product-title}
+
+This section introduces how to install mirror registry for {product-title}, a small and streamlined container registry to use as a target for mirroring the required container images of {product-title} for disconnected installations.
+
+.Prerequisites
+
+* An {product-title} subscription.
+* {op-system-base-full} 8 or Fedora CoreOS with Podman 3.3 installed.
+* About 12 GB for {product-title} 4.10 Release images, or about 358 GB for {product-title} 4.10 Release images and {product-title} 4.10 Red Hat Operator images.
```

Comment:
Can we explain why this recommendation is the way it is? 
Is the 12 GB for the Release Payload and Related Images; for 1 version (x.y.z) or does this cover N (releases, z streams)? 

I think we need to elaborate on how we 'arrived' at these numbers, as these statement (as is) just invite questions (or a support burden).